components/studio/bin/hab-studio.sh

```diff
@@ -712,8 +712,23 @@ rm_studio() {
 
   cleanup_studio
 
-  # Remove remaining filesystem
-  $bb rm -rf $v "$HAB_STUDIO_ROOT"
+  if $bb mount | $bb grep -q "on ${HAB_STUDIO_ROOT}.* type"; then
```

Comment:
Would it be possible to augment`is_fs_mounted` to be able to use it here, too?

Just thinking of ways to consolidate the core logic in one place.

Reply:
I noticed that as well and thought the best I could do was to check for `grep -q "on ${HAB_STUDIO_ROOT}/.* type"` (notice the directory slash) which could allow for `if is_fs_mounted '.*'; then …; fi` although that felt a bit aggressive too. This current case additionally should be able to detect if the `$HAB_STUDIO_ROOT` path itself was mounted (why in the world would this be mounted you ask? I don't know but I figured in the pathological case, it was worth checking).